docs/features/distribution.md

```diff
@@ -2,8 +2,17 @@
 
 ## Introduction 
 
-This module is in charge of distributing collected transaction fee and inflated token to all validators and delegators. To reduce computation stress, a lazy distribution strategy is brought in. `lazy` means that the benefit won't be paid directly to contributors automatically. The contributors are required to explicitly send transactions to withdraw their benefit, otherwise, their benefit will be kept in the global pool. 
+This module is in charge of distributing collected transaction fee and inflated token to all validators and delegators. 
+To reduce computation stress, a lazy distribution strategy is brought in. 
+`lazy` means that the benefit won't be paid directly to contributors automatically. 
+The contributors are required to explicitly send transactions to withdraw their benefit, otherwise, 
+their benefit will be kept in the global pool. 
 
+If one validator is the proposer of current round, that validator (and their delegators) receives between 1% and 5% of fee rewards as proposer reward.
```

Comment:
receives between 1% and 5% of the sum of fee rewards and inflation token as proposer reward.